topics/covid19/current-patterns-transmission/_posts/2020-03-22-ICU-projections.md

```diff
@@ -125,9 +110,30 @@ Assuming the proportion of COVID-19 cases needing critical care remains constant
 <br>
 
 
+## Regional Results {.tabset .tabset-fade .tabset-pills}
+
+### NHS regions (excl. London)
+
+For the analysis excluding London, the starting point for admissions projections is taken from the *18th March 2020*.
+
+<img src="figures/ICU-projections-excLondon.png" width="75%"> <br>
+**Figure 2: Predictions of critical care bed needs - NHS regions excluding London.** This figure shows the numbers of new patients requiring beds in critical care for COVID-19 projected from current admissions data in ICU/HDU in England. Mean needs and their 95% confidence intervals are indicated by the plain lines and ribbons, respectively. Columns and colors present results for different assumed reporting of admissions, from full reporting (100%, left), 90% (middle), and 80% (right) reporting. Rows indicate results for different assumed distributions of the duration of hospitalisation: 'short' (median: 8 days<sup>2</sup>) and 'long' (median: 10 days<sup>3</sup>).
+
+<br>
+
+### London
+
+For the analysis of London specifically, the starting point for admissions projections is taken from the *20th March 2020*.
```

Comment:
Average demand ranging from xxx (CI95%: xxx ; xxx) to xxx (CI95%: xxx ; xxx) critical care beds every day on the 31st March 2020.